css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,30 @@
+input:hover {
+  background-color: black;
+  color: white;
+  cursor: pointer;
+}
+
+table {
+  table-layout: fixed;
+  border: 2px solid white;
+}
+
+thead th:nth-child(1) {
+  background-color: black;
+  color: white;
+}
+
+thead th:nth-child(2) {
+  background-color: black;
+  color: white;
+}
+
+thead th:nth-child(3) {
+  background-color: black;
+  color: white;
+}
```

Comment:
Since each of these rules has the same declaration block, you should just do `thead th` and apply styling to all the `th` elements in one rule